src/request/transaction-report.ts

```diff
@@ -10,9 +10,12 @@ interface TransactionReportProps {
    */
   chargebackCode?: string;
   /**
-   * The IP address of the customer placing the order.
+   * The IP address of the customer placing the order. This field is not
+   * required if you provide at least one of the transaction's minfraud_id,
+   * maxmind_id, or transaction_id. You are encouraged to provide it, if
```

Comment:
Maybe the casung should match the names in this interface (which also match the names used elsewhere in this package).